src/sentry/user_feedback/blueprints/api.md

```diff
@@ -0,0 +1,272 @@
+# User Feedback API
+
+Host: https://sentry.io/api/0
+
+**Authors.**
+
+@cmanallen
+
+**How to read this document.**
+
+This document is structured by resource with each resource having actions that can be performed against it. Every action that either accepts a request or returns a response _will_ document the full interchange format. Clients may opt to restrict response data or provide a subset of the request data. The API may or may not accept partial payloads.
+
+## User Feedback Index [/organizations/<organization_slug>/user-feedback/]
+
+- Parameters
```

Comment:
@colton should `cursor` be in here?

Reply:
Also, `utc  (optiona, "true" | "false")`

Reply:
Added cursor.  Don't understand the utc parameter idea.

Reply:
`utc` seems like a common param in other endpoints, to help with `start` and `end` when they're present? idk really, but the defaults param parsers might be helpful here

Reply:
we use the default so ill add it